great_expectations_cloud/agent/agent.py

```diff
@@ -535,7 +546,7 @@ def _set_http_session_headers(
             extra={
                 "user_agent": header_name.USER_AGENT,
                 "agent_version": agent_version,
-                "job_id": header_name.AGENT_JOB_ID,
```

Comment:
This is the only place that the key is being changed from `job_id` to `header_name`.  I think that is ok because

1) this is a log message, and this is the `extra` field of the message. 
2) IT is actually just a [enum](https://github.com/great-expectations/cloud/blob/bb06e6319b7300ff7bdb628caddaf500dfdd2702/great_expectations_cloud/agent/constants.py#L9) which is `Agent-Job-Id`